tools/mason/MasonUpdate.chpl

```diff
@@ -165,7 +165,7 @@ proc updateRegistry(tf: string, args: list(string)) {
       const localRegistry = registryHome;
       mkdir(localRegistry, parents=true);
       const cloneRegistry = 'git clone -q ' + registry + ' .';
-      writeln("Updating mason-registry");
+      writeln("Updating local-registry");
```

Comment:
Each registry directory has its own name. This change will require:

1. Getting that registry name in a variable
2. Printing that registry name here, instead of `mason-registry`

Reply:
> 2\. Printing that registry name here, instead of `mason-registry`

Ohh! Seems like I misunderstood the issue. I will update the changes accordingly :+1:

Reply:
@ben-albrecht I think we already have the registry name as `registry`, we just need to print it?

Reply:
oh, possibly. Maybe this is still an easy fix :)

Reply:
I have made the changes. Please have a look. :smile: